dockerfiles/run/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+
+services:
+
+  james:
+    depends_on:
+      - elasticsearch
+      - cassandra
+    entrypoint: bash -c "java -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=8765 -Dworking.directory=/root/ -jar james-server.jar"
```

Comment:
maybe we could handle the debug part into the james Dockerfile and enable it only when env DEBUG is set to 1 ?